src/Avalonia.Controls/RowDefinition.cs

```diff
@@ -108,5 +110,32 @@ public GridLength Height
         internal override GridLength UserSizeValueCache => this.Height;
         internal override double UserMinSizeValueCache => this.MinHeight;
         internal override double UserMaxSizeValueCache => this.MaxHeight;
+
+        /// <inheritdoc/>
+        public bool Equals(RowDefinition other)
+        {
+            return other != null
+                ? Hash(this) == Hash(other)
```

Comment:
And here.

Reply:
sorry, had problems with my git config...